wagtailio/core/blocks.py

```diff
@@ -374,3 +374,165 @@ class IconBulletsBlock(blocks.StructBlock):
     class Meta:
         # template = "" # TODO: add template
         label = "Icon bullets"
+
+
+class StandaloneQuoteBlock(blocks.StructBlock):
+    quote = blocks.TextBlock(required=True)
+    author = blocks.RichTextBlock(
```

Comment:
Frankly this doesn't make lots of sense based on the designs. This should at most allow link, but best to make it a CharBlock

Reply:
Indeed, I've removed all* the features in f3751fde600de09043686218a674a322b0039046.

\* except `link`, since there'd probably be a need to link to the author's homepage/tweet/etc.